pqcrypto-dilithium/README.md

```diff
@@ -31,6 +31,15 @@ methods only.
   * ``clean``
   * ``avx2`` (if supported)
   * ``aarch64`` (if supported)
+* ``dilithium2aes``
+  * ``clean``
+  * ``avx2`` (if supported)
+* ``dilithium3aes``
+  * ``clean``
+  * ``avx2`` (if supported)
+* ``dilithium5aes``
+  * ``clean``
+  * ``avx2`` (if supported)
```

Comment:
Strictly speaking we should also check if the CPU has AES support... but I think we'll just overlook that for now, as I'd need to patch the generator.